packages/node-api/src/interfaces/augment-api-tx.ts

```diff
@@ -21,112 +21,112 @@ declare module '@polkadot/api-base/types/submittable' {
     assets: {
       /**
        * Approve an amount of asset for transfer by a delegated third-party account.
-       * 
+       *
```

Comment:
It would be nice to prevent such automatic style fixes by your IDE on the files that are automatically generated. It generates commits which larger than necessary.

Reply:
Yes indeed this kind of auto-format is not really handy on files that are generated again and again. Done.